configs/webserver/halfstack.conf

```diff
@@ -33,6 +33,8 @@ max_file_upload_size = 4294967296
 
 [pipeline]
 endpoint = "http://127.0.0.1:9500"
+frontend-endpoint = "http://127.0.0.1:3000"
```

Comment:
```suggestion
endpoint = "http://127.0.0.1:3000"
```

Reply:
Thanks for the feedback! This was intended - `endpoint` has to be preserved since is has been already used in many places. Instead, this PR is to **add** new configs than to replace one.